oracledb-ballerina/Config.toml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+[sql]
+maxOpenConnections=10
+maxConnectionLifeTimeInSeconds=2000.5
+minIdleConnections=5
```

Comment:
With the new changes, we need to move this file to the test directory to use it in test cases. `Config.toml` inside the module doesn't read.

Reply:
resolved with 171baaf97be39b48c6663ec8edeb3d05944f98f0